roles/ovirt-repositories/tasks/rh-subscription.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Register and subscribe to multiple pools.
+  redhat_subscription:
```

Comment:
Does this also enable the repos?

Reply:
Are we suppose to have configurable URLs?

Reply:
1) When you specify the pool you should get the repositories related to the pool and all non optional repositories should be enabled.
At least that what I get from this DOC: https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html-single/rhsm/index#supplementary-repos

Reply:
2. You mean being able to bring custom repositories? I think it's not in the scope of this role from what I understand.

Reply:
1 .I am unsure if rhv provides any optional repo, so hopefully this is enough. 

2. Custom URL for testing servers for example.

Reply:
2. We can leave it up for future enhancements I guess.